magenta/models/music_vae/data.py

```diff
@@ -454,8 +457,11 @@ def _dedupe_and_sample(self, all_sliced_lists):
     # TODO(adarob): Consider handling the fact that different event lists can
     # be mapped to identical tensors by the encoder_decoder (e.g., Drums).
 
-    unique_multievent_tuples = list(set(tuple(l)
-                                        for l in sliced_multievent_lists))
+    if self._dedupe_event_lists:
+      unique_multievent_tuples = list(set(tuple(l)
+                                          for l in sliced_multievent_lists))
+    else:
+      unique_multievent_tuples = list(sliced_multievent_lists)
```

Comment:
Do you still need to convert the individual lists to tuples?

Also, should maybe just call this multievent_tuples since it's no longer guaranteed to be unique.